Client/Clique/Modules/Profile/ProfileEditViewController.swift

```diff
@@ -0,0 +1,128 @@
+        emailTextField.isSkeletonable = true
+        emailTextField.showAnimatedSkeleton(usingColor: .clouds, transition: .crossDissolve(0.5))
+        
+        nameTextfield.layer.masksToBounds = false
+        nameTextfield.isSkeletonable = true
+        nameTextfield.showAnimatedSkeleton(usingColor: .clouds, transition: .crossDissolve(0.5))
+        
+        surnameTextField.layer.masksToBounds = false
+        surnameTextField.isSkeletonable = true
+        surnameTextField.showAnimatedSkeleton(usingColor: .clouds, transition: .crossDissolve(0.5))
+        
+        textViewBio.layer.masksToBounds = false
+        textViewBio.isSkeletonable = true
+        textViewBio.showAnimatedSkeleton(usingColor: .clouds, transition: .crossDissolve(0.5))
+        
+        datePicker.layer.masksToBounds = false
+        datePicker.isSkeletonable = true
+        datePicker.showAnimatedSkeleton(usingColor: .clouds, transition: .crossDissolve(0.5))
+        
+        SkeletonAppearance.default.skeletonCornerRadius = 100
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        getUser()
+    }
+    
+    private func getUser() {
+        profileService.getUser { result in
+            switch result {
+            case .success(let user):
+                    self.nameTextfield.text = user.name
+                    self.surnameTextField.text = user.surname
+                    self.emailTextField.text = user.email
+                    self.textViewBio.text = user.bio
+                    self.imgProfile.stopSkeletonAnimation()
+                    self.emailTextField.stopSkeletonAnimation()
+                    self.nameTextfield.stopSkeletonAnimation()
+                    self.surnameTextField.stopSkeletonAnimation()
+                    self.textViewBio.stopSkeletonAnimation()
+                    self.datePicker.stopSkeletonAnimation()
+                    
+                    self.view.hideSkeleton(reloadDataAfter: true,
+                                           transition: .crossDissolve(0.5))
+            case .failure:
+                return
+            }
+        }
+    }
+    
+    @IBAction func saveButtonPressed(_ sender: UIButton) {
+        guard let userProfileData = getProfileData() else {
+            Constants.Alerts.alert(fwdMessage: Constants.Alerts.pleaseEnterInfoMsg, viewController: self)
+            return
+        }
+        
+        updateUser(with: userProfileData)
+        dismiss(animated: true, completion: nil)
+    }
+    
+    func updateUser(with userUpdateData: UserProfileUpdateData) {
+        profileService.updateUser(with: userUpdateData) { (isSuccess) in
+            if isSuccess {
+                Constants.Alerts.alert(fwdMessage: Constants.Alerts.successfullyUpdatedMsg, viewController: self)
+            }else {
+                Constants.Alerts.alert(fwdMessage: Constants.Alerts.pleaseEnterInfoMsg, viewController: self)
```

Comment:
You also need to use weak self here
```suggestion
let message = isSuccess ? Constants.Alerts.successfullyUpdatedMsg : Constants.Alerts.pleaseEnterInfoMsg
Constants.Alerts.alert(message: message, viewController: self)
```